### PR TITLE
Create Executive vote - July 06, 2022.md

### PR DESCRIPTION
**Pull request by blimpa**
Exec copy July 06, 2022: Core Unit budget distributions for GRO-001 and SF-001; Recognized Delegate compensation for June 2022; Keep3r Keeper Network stream setup.

@patrick-j-govalpha, @prose11

### CODE REVIEW

File: governance/votes/Executive vote - July 06, 2022.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+#### Core Unit Budget Transfers
+
+- **648,133.59  DAI** will be transferred to the [Growth Core Unit (GRO-001)](https://mips.makerdao.com/mips/details/MIP39c2SP4) at [0x7800C137A645c07132886539217ce192b9F0528e](https://etherscan.io/address/0x7800C137A645c07132886539217ce192b9F0528e) as a result of [their most recent budget approval](https://vote.makerdao.com/polling/QmdMkg2N).
+
+
+### Recognized Delegate Compensation
+
+As per [MIP61: Recognized Delegate Compensation](https://mips.makerdao.com/mips/details/MIP61), a total of **85,185 DAI** will be distributed to 14 Recognized Delegates as itemized below if this executive proposal passes:
+
+- **11573 DAI** transferred to [Feedblack Loops LLC](https://vote.makerdao.com/address/0x845b36e1e4f41a361dd711bda8ea239bf191fe95) at [0x80882f2A36d49fC46C3c654F7f9cB9a2Bf0423e1](https://etherscan.io/address/0x80882f2A36d49fC46C3c654F7f9cB9a2Bf0423e1).
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - July 06, 2022.md:52]: Add comma for consistency with other entries

```suggestion
- **11,573 DAI** transferred to [Feedblack Loops LLC](https://vote.makerdao.com/address/0x845b36e1e4f41a361dd711bda8ea239bf191fe95) at [0x80882f2A36d49fC46C3c654F7f9cB9a2Bf0423e1](https://etherscan.io/address/0x80882f2A36d49fC46C3c654F7f9cB9a2Bf0423e1).
```

File: governance/votes/Executive vote - July 06, 2022.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+- **11,528 DAI** transferred to [Flip Flop Flap Delegate LLC](https://vote.makerdao.com/address/0xaf8aa6846539033eaf0c3ca4c9c7373e370e039b) at [0x688d508f3a6B0a377e266405A1583B3316f9A2B3](https://etherscan.io/address/0x688d508f3a6B0a377e266405A1583B3316f9A2B3).
+- **11,292 DAI** transferred to [schuppi](https://vote.makerdao.com/address/0xb21e535fb349e4ef0520318acfe589e174b0126b) at [0x89C5d54C979f682F40b73a9FC39F338C88B434c6](https://etherscan.io/address/0x89C5d54C979f682F40b73a9FC39F338C88B434c6).
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - July 06, 2022.md:54]: This is an outdated address for schuppi - guessing you copy-pasted from a previous exec?

```suggestion
- **11,292 DAI** transferred to [schuppi](https://vote.makerdao.com/address/0xb21e535fb349e4ef0520318acfe589e174b0126b) at [0xCCffDBc38B1463847509dCD95e0D9AAf54D1c167](https://etherscan.io/address/0xCCffDBc38B1463847509dCD95e0D9AAf54D1c167).
```

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]

File: governance/votes/Executive vote - July 06, 2022.md (hunk 2 above)
Comment (blimpa) [reply governance/votes/Executive vote - July 06, 2022.md:54]: Correct *shame*

**Comment by patrick-j-govalpha** [comment]
@prose11 Governance House address looks good to me, happy for merge if you're happy with the rest of the copy :thumbsup:

**Review by prose11 — APPROVED** [review]
Nice work fellas :)

**Pull request merged**